docs/user_guide/HOWTO/Manage_TPM.rst

```diff
@@ -72,9 +74,90 @@ not necessarily preclude a TPM, as a software TPM may be present.
       reserved2 => "clear"
       ...
 
+TPM Simulators
+--------------
 
-TPM 1.2
-^^^^^^^
+The SIMP TPM 1.2 and TPM 2.0 simulator RPMs are packages containing
+compiled versions of IBM's Software TPMs. The RPMs can be regenerated by
+downloading the code from their respective project repositories, `TPM 1.2`_ and 
+`TPM 2.0`_. After installing the RPMs, the TPMs can be initialized.
+
+  .. NOTE::
+      Root permission, either via sudo or running as root, is necessary to
+      install and initialize the TPM simulators.
+
+Installing and Initializing the TPM 1.2 Simulator
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The TPM 1.2 simulator RPM is included with the SIMP Extras packages and can
+be installed via yum with the command:
+
+   .. code-block:: bash
+
+     # yum install simp-tpm12-simulator
+
+
+To initalize the TPM 1.2 simulator on EL6, issue the following commands:
+
+   .. code-block:: bash
+
+     # service tpm12-simulator start
+     # service tpm12-tpmbios start
+     # service tpm12-simulator restart
+     # service tpm12-tpmbios start
+     # service tpm12-tpminit start
+     # service tpm12-tcsd start
+
+To initalize the TPM 1.2 simulator on EL7, issue the following commands:
+
+   .. code-block:: bash
+
+     # systemctl start tpm12-simulator
+     # systemctl start tpm12-tpmbios
+     # systemctl restart tpm12-simulator
+     # systemctl start tpm12-tpmbios
+     # systemctl start tpm12-tpminit
+     # systemctl start tpm12-tcsd
+
+The `tpm_version` command provided by tpm-tools RPM can be used to verify the 
+TPM 1.2 simulator has been initialized.
+
+   .. code-block:: bash
+
+     # tpm_version
+
+Installing and Initializing the TPM 2.0 Simulator
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The TPM 2.0 simulator RPM is included with the SIMP Extras packages (EL7 only)
+and can be installed via yum with the command:
+
+   .. code-block:: bash
+
+      # yum install simp-tpm2-simulator
+
+
+To initalize the TPM 2.0 simulator, issue the following commands:
+
+   .. code-block:: bash
+
+     # runuser tpm2sim --shell /bin/sh -c "cd /tmp; nohup \
+       /usr/local/bin/tpm2-simulator &> /tmp/tpm2-simulator.log &"
+     # mkdir -p /etc/systemd/system/tpm2-abrmd.service.d
+     # printf "[Service]\nExecStart=\nExecStart=/sbin/tpm2-abrmd -t socket" \
+       > /etc/systemd/system/tpm2-abrmd.service.d/override.conf
+     # systemctl daemon-reload
+     # systemctl start tpm2-abrmd
+
+The `tpm2_getcap` command provided by tpm2-tools RPM can be used to verify the 
+TPM 2.0 simulator has been initialized:
+ 
+   .. code-block:: bash
+
```

Comment:
- Please add ``yum install tpm2-tools``
- Might be nice to tell the user what the expected output looks like.

Reply:
Add tpm2-tools installation and sample command output to the Starting with TPM section.

Reply:
?  Didn't see the change.